yellow-paper/docs/decentralisation/governance.md

```diff
@@ -74,19 +64,17 @@ Rollup instances themselves will need to deposit their stake into the governance
 #### Results
 If the vote fails, there is no action needed.
 
-If the vote passes, and a new rollup has been determined to be the next canonical instance, it will become canonical in the amount of days defined within the vote's timelock. It is likely there are defined limitations around this parameter, e.g.,it must be a 3-30 day timelock. This is explained more in the timing section below. At this block height, portals that desire to follow governance should start referencing the new canonical instance to ensure as many bridged assets are backed on the latest version as possible.
+If the vote passes, and a new rollup has been determined to be the next canonical instance, it will become canonical in the amount of days defined within the vote's timelock. It is likely there are defined limitations around this parameter, e.g., it must be a 3-30 day timelock. This is explained more in the timing section below. At this block height, portals that desire to follow governance should start referencing the new canonical instance to ensure as many bridged assets are backed on the latest version as possible.
 
 :::danger
-Question: what is needed to pass a vote? 
+**Question**: what is needed to pass a vote? 
 
 Current thinking is that it should likely be the amount expected to be held in the initial instance of the rollup, e.g. 20% circulating & 25% of that is staked -> 5% of total supply, so locked tokens do not need to participate whatsoever in a happy path upgrade to v1.
 
-:::warning
-This needs to be clarified.
-:::
 
-:::danger
-Lasse: note that if the portals follow the governance registry blindly, they assume that new inbox/outbox will always be backwards compatible
+**Lasse**: 
+- This needs to be clarified. I don't understand the sentence.
```

Comment:
Sorry for the confusion. I think my intention ws to say that the "minimum quorum" required to pass a vote should be a customizable parameter that we specifically determine the value of via a simulation or modeling and further analysis. My initial feeling for what this minimum quorum parameter would be the amount of stake that is expected to be held in the initial instance of the rollup, so that they are able to pass votes without any other ecosystem participants, such as locked token holders needing to participate. So (hypothetically) within year 1, if 20% of total supply is expected to be circulating, and it's expected that 25% of those tokens are staked to the current canonical rollup, then the quorum threshold for passing a governance vote should be ~5% of total supply. Which _feels_ reasonable. Does that make sense?

Reply:
I think the notation of saying a minimum quorum to pass the vote is confusing here. e.g., indicates that 5% and you can pass whatever you want. 
It is practically a quorum before a proposal can successfully be made. Meaning that below 5% no upgrades are possible, we cannot even get to vote on them.